constitution.md

```diff
@@ -0,0 +1,196 @@
+# Constitution of the Unix Users Group
+
+October 25, 1999  
+Authored by David Wilburn,  
+Amended, May 1, 2003.  
+Amended, April 1, 2019.  
+
+## Article I - Name of Organization
+
+This organization shall be called the James Madison University Unix Users
+Group. The Unix Users Group is also referred to as the UUG.
+
+## Article II - Purpose of Organization
+
+This organization is devoted to users of the Unix computer operating system.
+Its mission:
+- To promote Unix awareness and knowledge, and
+- To provide a means of communication between persons having interest in the
+Unix operating system.
+
+## Article III - Membership
+
+Membership in this organization is open to all JMU students and will not be
+restricted on the basis of age, ability, ethnicity, gender, national origin,
+race, color, religion, national origin, veteran status, sexual orientation,
+gender identity, or political affiliation except in circumstances outlined in
+federal and state laws.
+
+Students in good standing with the University (2.0 cumulative G.P.A.) are
+eligible for membership after attending or participating in an organization
+event/meeting. As a member, one is required to attend organization meetings
+regularly, pay dues if required, and actively support organization projects.
+Membership will be revoked by ½ vote of officers plus ¾ vote from the general
```

Comment:
Once we determine what it means to be a member, we also need to determine what it means to lose membership.